native/c/zds.cpp

```diff
@@ -614,8 +625,20 @@ int zds_list_data_sets(ZDS *zds, string dsn, vector<ZDSEntry> &attributes)
           entry.volser = value;
         }
       }
+#define MIGRAT_VOLUME "MIGRAT"
+#define ARCIVE_VOLUME "ARCIVE"
+
+      if (entry.volser == MIGRAT_VOLUME || entry.volser == ARCIVE_VOLUME)
```

Comment:
I like this implementation 🥳 
This makes it flexible enough to support some third-party mainframe product that uses other conventions for migrating/archiving content. 🙏